build/scripts/LoadNuGetInfo.cmd

```diff
@@ -2,11 +2,10 @@
 
 REM This is a script that will set environment information about where to find
 REM NuGet.exe, it's version and ensure that it's present on the enlistment.
-set NuGetExeVersion=3.6.0-beta1
 set NuGetExeFolder=%~dp0..\..
```

Comment:
This file only exists at this point to serve roslyn-internal.  Otherwise I'd nuke it entirely.  I can't change anything here without it being a cross cutting violation.